### PR TITLE
Remove unused feature flags on ios

### PR DESCRIPTION
**Pull request by moderateepheezy**
# 📲 What

Removed every instance and implementation for Qualtrics flag in the codebase.

# 🤔 Why
Qualtrics implementation is currently redundant and currently has no use the app.

### CODE REVIEW

**Comment by nativeksr** [comment]
### SwiftFormat found issues:

| File | Rules |
| ---- | ----- |
| Library/Extensions/Feature+HelpersTests.swift | blankLinesAtStartOfScope |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Kickstarter.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7119,6 +7118,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEFINES_MODULE = YES;
+				DEVELOPMENT_TEAM = 48YBP49Y5N;
```
Comment (jgsamudio) [inline Kickstarter.xcodeproj/project.pbxproj:7121]: Do you know what caused this change? Should the development team be under the group above?

Comment (moderateepheezy) [reply Kickstarter.xcodeproj/project.pbxproj:7121]: I don't know the cause of the change.

File: Library/Extensions/Feature+HelpersTests.swift [hunk 2]
```diff
@@ -5,31 +5,6 @@ import Prelude
...
-    withEnvironment(config: config) {
-      XCTAssertFalse(featureQualtricsIsEnabled())
-    }
-  }
-
-  func testFeatureQualtrics_isFalse_whenNil() {
-    withEnvironment(config: .template) {
-      XCTAssertFalse(featureQualtricsIsEnabled())
-    }
-  }
```
Comment (singhhari) [inline Library/Extensions/Feature+HelpersTests.swift:32]: Just pinged you about the formatter. Give that a run so it fixes spacing issues.

**Review by singhhari — CHANGES_REQUESTED** [review]

**Comment by nativeksr** [comment]
### SwiftFormat found issues:

| File | Rules |
| ---- | ----- |
| Library/Extensions/Feature+HelpersTests.swift | blankLinesAtStartOfScope |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by singhhari — APPROVED** [review]
lgtm

**Pull request merged**